app/components/QuestionForm/PostRules.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { css } from '@emotion/react';
+
+import { BG_PRIMARY, RULES_BACKGROUND, TEXT_SECONDARY } from 'style-constants';
+
+import Label from 'components/FormFields/Label';
+import { FULL_RULES_LINK } from 'app/constants/rules';
+import { singleCommunityColors } from 'utils/communityManagement';
+
+const colors = singleCommunityColors();
+
+const messagesArray = [
+  'common.contentPopupBlock_3',
+  'common.contentPopupBlock_4',
+  'common.contentPopupBlock_5',
+  'common.contentPopupBlock_6',
+  'common.contentPopupBlock_7',
+];
+
+const PostRules = () => {
+  const { t } = useTranslation();
+
+  return (
+    <div
+      css={css`
+        margin-bottom: 32px;
```

Comment:
put all styles in a separate object and use the properties of this object in the component

Reply:
Done